src/index.ts

```diff
@@ -1,8 +1,9 @@
-import { Union as _Union } from './union';
+import { Union as _Union, VolOptions } from './union';
 import { IFS } from './fs';
+export * from './lists'
```

Comment:
I'm not too keen on making these part of the public API, as they're more to maintain, and once public they can't be changed without a breaking release.

(This was a problem the original lists provided by `fs-monkey` were already starting to suffer from - in fact `fs-monkey` is still actually waiting for a new release to fix a typo I made).

You can still import these into your own project should you wish to use them, with minimal extra overhead.